geomesa-hbase/geomesa-hbase-tools/src/main/resources/org/locationtech/geomesa/hbase/tools/hbase-libjars.list

```diff
@@ -1,5 +1,8 @@
 hbase-client
 hbase-common
+hbase-hadoop-compat
+hbase-hadoop2-compat
+hbase-metrics
```

Comment:
Did you test this?  (Or is their addition just obvious?)

Reply:
i had to add these when running the hbase offline ingest. we don't normally smoke test that (we probably should start), so it's probably been broken for a while.

Reply:
+1 to both!  Can you upload the smoke test?

Reply:
updated the checklist